src/server/server.ts

```diff
@@ -22,119 +26,125 @@ import { App, Stack, Stage } from './appIdentity';
+                            logger.error(error.message);
+                            res.sendStatus(500);
+                        }
+                    },
+                    content => {
+                        const {
+                            resources,
+                            element,
+                            hydrationProps,
+                        } = Page({ content, imageSalt, getAssetLocation });
+                        res.set('Link', getPrefetchHeader(resources));
+                        res.write('<!DOCTYPE html>');
+                        res.write('<meta charset="UTF-8" />');
+                        res.write(`<script id="hydrationProps" type="application/json">${JSON.stringify(hydrationProps)}</script>`);
+                        res.write(renderToString(element));
+                        res.end();
+                    }
+                )
             }
-          },
-          content => {
-            const {
-              resources,
-              element,
-              hydrationProps
-            } = Page({ content, imageSalt, getAssetLocation });
-            res.set('Link', getPrefetchHeader(resources));
-            res.write('<!DOCTYPE html>');
-            res.write('<meta charset="UTF-8" />')
-            res.write(`<script id="hydrationProps" type="application/json">${JSON.stringify(hydrationProps)}</script>`)
-            res.write(renderToString(element));
-            res.end();
-          }
-        )
-      }
-    } else {
-      const response: ErrorResponse = ErrorResponse.read(protocol);
-      logger.error(`I received a ${capiResponse.status} code from CAPI with the message: ${response.message} for resource ${articleId}`);
-      res.sendStatus(500);
+        } else {
+            const response: ErrorResponse = ErrorResponse.read(protocol);
+            logger.error(`I received a ${capiResponse.status} code from CAPI with the message: ${response.message} for resource ${articleId}`);
+            res.sendStatus(500);
+        }
+    } catch (e) {
+        logger.error(`This error occurred`, e);
+        res.sendStatus(500);
     }
-  } catch (e) {
-    logger.error(`This error occurred`, e);
-    res.sendStatus(500);
-  }
 }
 
+
 // ----- App ----- //
+
 logger.info(`Starting ${App} in ${Stage} for the stack ${Stack}`);
+
 if (process.env.NODE_ENV === "production") {
-  logger.info("Node is running in production mode")
+    logger.info("Node is running in production mode");
 }
-const app = express();
-app.use(bodyParser.raw({limit: '50mb'}));
-
-app.all('*', (request, response, next) => {
-  const start = Date.now();
-  response.once('finish', () => {
-    const duration = Date.now() - start;
-    logger.info(`HTTP ${request.method} ${request.path} returned ${response.statusCode} in ${duration}ms`)
-  });
 
-  next();
-});
+const app = express();
 
+app.use(bodyParser.raw({ limit: '50mb' }));
 app.use('/assets', express.static(path.resolve(__dirname, '../assets')));
```

Comment:
I thought this was used to serve assets locally?

Reply:
Currently it serves the fonts locally, yes.